task/backend/executor/executor.go

```diff
@@ -378,7 +378,7 @@ func (p *asyncRunPromise) finish(res *runResult, err error) {
 		} else if res.err != nil {
 			p.logger.Info("Got result with error", zap.Error(res.err))
```

Comment:
Probably someone from tasks can answer, but why is this one `Info` while the other "Got result with error" is `Debug`? Anyway, it's as it is before so no reason to block this change, just was curious.

Reply:
Totally agreed, but to your point I'm just restoring to previous condition. I think this is a known configuration for `tigerteam`